lib/elixir/src/elixir_module.erl

```diff
@@ -199,6 +200,9 @@ validate_on_load_attribute({on_load, Def}, Defs, File, Line) ->
   end;
 validate_on_load_attribute(false, _Defs, _File, _Line) -> ok.
 
+is_behaviour(DataBag) ->
+  ets:member(DataBag, {accumulate, callback}) orelse ets:member(DataBag, {accumulate, callback}).
```

Comment:
Am I going crazy or are both sides of `orelse` the same? :D

Reply:
Are the same thing